lib/shopify_api/graphql/response.ex

```diff
@@ -37,6 +37,8 @@ defmodule ShopifyAPI.GraphQL.Response do
   def handle({:error, _} = response, _), do: response
 
   @doc false
+  def build_response(%{body: %{"data" => nil}} = response), do: {:error, response}
```

Comment:
👍

Reply:
🍎 might be nice to have a comment in here to let us know when this condition happens